system/handlers/renderers/content/auditLogEntry.cfc

```diff
@@ -0,0 +1,33 @@
+component  {
+
+	property name="AuditService" inject="AuditService";
+
+	private string function default( event, rc, prc, args={} ) {
+		var auditTrailId  = args.id ?: ""
+		var records       = AuditService.getAuditTrail( auditTrailId );
+
+		return renderView( view="/renderers/content/auditLogEntry/default", args=args );
+	}
+
+	private string function login_success( event, rc, prc, args={} ) {
+		var auditTrailId  = args.id ?: ""
+		var records       = AuditService.getAuditTrail( auditTrailId );
```

Comment:
rather than `records`, this should be named `auditLog` (we expect a single audit log record here)